netdata-installer.sh

```diff
@@ -1091,14 +1099,15 @@ if [ ! -f "${NETDATA_PREFIX}/etc/netdata/.installer-cleanup-of-stock-configs-don
           # it is a stock version - remove it
           echo >&2 "File '${TPUT_CYAN}${x}${TPUT_RESET}' is stock version of '${t}'."
           run rm -f "${x}"
+	  # shellcheck disable=SC2030
```

Comment:
Indenting mismatch here (you’ve got a mix of tabs and spaces in the indentation for this line).